tools/travis.sh

```diff
@@ -1,7 +1,10 @@
 set -x
 ulimit -n 4096
+# do this for our daily build test run
+if [[ "$TRAVIS_EVENT_TYPE" == "cron" ]]; then
```

Comment:
yah - that's the plan for the future, but I don't know what that hook will look like. For the moment I wanted to get builds started sooner, rather than later.